tests/console/consoletest_setup.pm

```diff
@@ -66,28 +66,18 @@ sub run() {
     select_console 'root-console';
 
     type_string "chown $username /dev/$serialdev\n";
-    # Export the existing status of running tasks and system load for future reference (fail would export it again)
-    script_run "ps axf > /tmp/psaxf.log";
-    script_run "cat /proc/loadavg > /tmp/loadavg_consoletest_setup.txt";
```

Comment:
They were duplicated in `export_logs()` I removed them from the test and left in the sub. Would you prefer to keep them duplicated?